libs/ledger-live-common/src/families/cosmos/helpers.ts

```diff
@@ -13,3 +13,27 @@ export const getMainMessage = (messages: CosmosMessage[]): CosmosMessage => {
     .sort((a, b) => messagePriorities.indexOf(a.type) - messagePriorities.indexOf(b.type));
   return sortedTypes[0];
 };
+
+export const sortObjectKeysDeeply = src => {
+  let out;
+
+  if (Array.isArray(src)) {
+    return src.map(function (item) {
+      return sortObjectKeysDeeply(item);
+    });
+  }
+
+  if (typeof src === "object") {
+    out = {};
+
+    Object.keys(src)
+      .sort((a, b) => a.localeCompare(b))
+      .forEach(function (key) {
+        out[key] = sortObjectKeysDeeply(src[key]);
+      });
+
+    return out;
+  }
+
+  return src;
+};
```

Comment:
I'm thinking we should set a safety limit on the number of recursions. A bit paranoid, but just to be safe, since JS doesn't forbid recursive objects.
We can put something large enough, like `20` or idk, just to prevent infinite loops.

Reply:
No longer relevant as we use cosmjs lib now